django/contrib/postgres/fields/jsonb.py

```diff
@@ -44,6 +44,13 @@ def __init__(self, verbose_name=None, name=None, encoder=None, **kwargs):
     def db_type(self, connection):
         return 'jsonb'
 
+    def get_sql_placeholder(self, quoted_name, key_lookups):
+        if not key_lookups:
+            return '{} = %s'.format(quoted_name)
+
+        keys = ','.join(key_lookups)
+        return "{name} = jsonb_set({name}, '{{{keys}}}', %s::jsonb)".format(name=quoted_name, keys=keys)
```

Comment:
Another thing to consider when using `jsonb_set`; what should be done when trying to update a value a `NULL` value's key?

https://www.2ndquadrant.com/en/blog/jsonb_set_lax/